src/wp-content/themes/twentytwenty/classes/class-twentytwenty-script-loader.php

```diff
@@ -33,10 +55,17 @@ class TwentyTwenty_Script_Loader {
 		 * @return string Script HTML string.
 		 */
 		public function filter_script_loader_tag( $tag, $handle ) {
-			foreach ( array( 'async', 'defer' ) as $attr ) {
-				if ( ! wp_scripts()->get_data( $handle, $attr ) ) {
-					continue;
-				}
+			$strategies = array(
+				'async' => (bool) wp_scripts()->get_data( $handle, 'async' ),
+				'defer' => (bool) wp_scripts()->get_data( $handle, 'defer' ),
+			);
+			$strategy   = wp_scripts()->get_data( $handle, 'strategy' );
+			if ( in_array( $strategy, array( 'async', 'defer' ), true ) ) {
```

Comment:
Nit-pick, and 0.0000001% faster :)
```suggestion
			if ( $strategy && isset( $strategies[ $strategy ] ) ) {
```

Reply:
I like it. Reusing a data structure we already have.